### ISSUE TITLE
Enhancement Request to add an option to scroll paths in the evidence modal

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#670]
I'd like a way to scroll through paths in the evidence modal instead of closing > path selection each time.

**Comment by gaurav** [comment]
Not a showstopper, but an enhancement request. Could be buttons or a scroll bar like thing. Problem: can't remember what path I just left, so switching from one to the other and be able to go back would be great.

**Comment by sierra-moxon** [comment]
@dnsmith124 - where does this rate in your priority list? :)

**Comment by dnsmith124** [comment]
@sierra-moxon to answer your question more concisely, it's currently not our highest priority, but is on our to-do list. It's a stretch goal of our current release plans.

**Comment by sstemann** [comment]
based on UI meeting today, I think this is happening in Guppy

**Comment by dnsmith124** [comment]
This feature is part of the UI facelift, but was deferred to the next sprint to accommodate other changes. Will update once this has been changed.

**Comment by sstemann** [comment]
it looks like this not in Hammerhead, just confirming @dnsmith124 , when I open a path, I see a plus/minus which isnt super clear what change this is and then i cant get to the next or previous path

This is the 5th path of 75 and I cant scroll to any other path in the modal. [MVP1 Diabetes Mellitus](https://ui.test.transltr.io/results?l=Diabetes%20Mellitus&i=MONDO:0005015&t=0&r=0&q=d7e7118c-4758-42de-933c-804b3fb6aa62)

![image](https://github.com/user-attachments/assets/99952468-9789-47e9-bfae-993290acc623)

**Comment by dnsmith124** [comment]
@sstemann that's correct, the ability to move between different paths from within the evidence modal will not be in Hammerhead. Reese has a design in the works that will resolve this in a more comprehensive way
